gpAux/gpdemo/demo_cluster.sh

```diff
@@ -247,8 +250,33 @@ if [ ! -x $GPPATH/gpinitsystem ]; then
     exit 1
 fi
 
+#*****************************************************************************************
+# Host configuration
+#*****************************************************************************************
+
+LOCALHOST=`hostname`
+
+if [ -n HOSTS_LIST  ]; then
+  rm -f hostfile
+  for host in $HOSTS_LIST; do
+    if ! ssh $host /bin/true; then
+      echo "can't access to the host $host, exiting"
+      exit 1
+    fi
+
+    echo $host >> hostfile
+  done
+  HOSTS_OPTS="-h hostfile"
+  TRUSTED_SHELL="ssh"
+else
+  echo $LOCALHOST > hostfile
+  HOSTS_OPTS=""
+  TRUSTED_SHELL="`pwd`/lalshell"
+fi
+
 if [ -d $DATADIRS ]; then
   rm -rf $DATADIRS
+  gpssh -f hostfile rm -rf $DATADIRS
```

Comment:
Do we need to remove DATADIRS on servers from hostfile, when $DATADIRS doesn't exist on local host?